client/src/app/core/services/time-travel.service.ts

```diff
@@ -0,0 +1,117 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { environment } from 'environments/environment';
+import { CollectionStringModelMapperService } from './collectionStringModelMapper.service';
+import { History } from 'app/shared/models/core/history';
+import { DataStoreService } from './data-store.service';
+import { WebsocketService } from './websocket.service';
+import { BaseModel } from 'app/shared/models/base/base-model';
+import { StorageService } from './storage.service';
+
+/**
+ * Interface for full history data objects.
+ * The are not too different from the history-objects,
+ * but contain full-data and a timestamp in contrast to a date
+ */
+interface HistoryData {
+    element_id: string;
+    full_data: BaseModel;
+    information: string;
+    timestamp: number;
+    user_id: number;
+}
+
+/**
+ * Service to enable browsing OpenSlides in a previous version.
+ *
+ * This should stop auto updates, save the current ChangeID and overwrite the DataStore with old Values
+ * from the servers History.
+ *
+ * Restoring is nor possible yet. Simply reload
+ */
+@Injectable({
+    providedIn: 'root'
+})
+export class TimeTravelService {
+    /**
+     * Determine if OpenSlides is running in History mode.
+     */
+    public inHistoryMode = false;
+
+    /**
+     * Constructs the time travel service
+     *
+     * @param http "Normal" Angular httpClient is required here, since the request must not end with "/"
```

Comment:
Hm, we might separate the path, query and fragments as different parameters. Would be nice if you can change this in this PR, or meight be done later (then open an issue). The HttpService should build up the correct url then. (See also analogon in the WebsocketService: There is a QueryDict defined and then builded to a complete url).